### PR TITLE
Add return type hints to `chunkedgraph`

### PR DESCRIPTION
**Pull request by bdpedigo**

### CODE REVIEW

**Review by fcollman — APPROVED** [review]

File: caveclient/chunkedgraph.py [hunk 1]
```diff
@@ -529,7 +528,7 @@ def execute_split(
         root_id,
         source_supervoxels=None,
         sink_supervoxels=None,
-    ):
+    ) -> tuple[int, list]:
```
Comment (fcollman) [inline caveclient/chunkedgraph.py:531]: this seems to be improper syntax in our testing environment

Comment (bdpedigo) [reply caveclient/chunkedgraph.py:531]: @fcollman what python versions are supposed to be supported? I can't actually find that information anywhere

I think this must be a python version thing; this works for me on 3.11. i have a vague memory of Tuple changing to tuple or something like that

Comment (bdpedigo) [reply caveclient/chunkedgraph.py:531]: https://github.com/seung-lab/CAVEclient/pull/143

Comment (bdpedigo) [reply caveclient/chunkedgraph.py:537]: Recommend merging the above first and rerunning tests for this one on all versions, but I think this should be fixed now

**Pull request merged**